docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md

```diff
@@ -41,19 +41,25 @@ To deploy a web application on Meshery, follow these steps:
 1. Navigate to the Configuration menu in Meshery.
 2. Select "Designs" from the menu.
 3. Click on the "Import Design" button.
+   
 [![Configuration menu]({{site.baseurl}}/assets/img/meshery-design/configuration-menu-design-import.png)]({{site.baseurl}}/assets/img/meshery-design/configuration-menu-design-import.png)
```

Comment:
Image has moved outside of the numbered list indentation.

Reply:
Great feedback, @sangramrath 🙌